src/Adapter/AdapterMessageConverter.php

```diff
@@ -0,0 +1,167 @@
+<?php
+/*
+ * This file is part of the prooph/event-store.
+ * (c) 2014-2015 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ * 
+ * Date: 8/14/15 - 8:06 PM
+ */
+
+namespace Prooph\EventStore\Adapter;
+
+use Prooph\Common\Messaging\Message;
+use Prooph\Common\Messaging\MessageConverter;
+use Prooph\Common\Messaging\MessageFactory;
+use Prooph\EventStore\Adapter\Exception\ConfigurationException;
+use Prooph\EventStore\Adapter\Exception\RuntimeException;
+
+/**
+ * Trait AdapterMessageConverter
+ *
+ * This trait adds message conversion capabilities to any event store adapter.
```

Comment:
Perhabs hint, that not all adapters need to serialize the payload (f.e. mongo db).